superset/config.py

```diff
@@ -1371,6 +1371,8 @@ def SQL_QUERY_MUTATOR(  # pylint: disable=invalid-name,unused-argument
 # Set to False to only allow viewing own recent activity
 ENABLE_BROAD_ACTIVITY_ACCESS = True
 
+DATAHUB_URL = "https://localhost:9002/"
```

Comment:
This looks like a value for the development environment. Are we good with leaving this here? I'm guessing it's meant to be overridden?